### ISSUE TITLE
"Forgot Password" > Invalid email format generates a 

### ISSUE THREAD

**Issue by MarleneMenser** [HTBox/crisischeckin#562]
**Steps**
1. Click on Forgot Password
2. Enter an invalid email format (i.e. "abc@") with quotation marks
https://crisischeckin-d.azurewebsites.net/Account/ForgotPassword
3. Click on Request New Password
4. The "Password reset request sent" page appears instead of an error message.
https://crisischeckin-d.azurewebsites.net/Account/PasswordResetRequested

![screenshot 2016-05-05 at 11 46 56 am](https://cloud.githubusercontent.com/assets/19208932/15048329/195ec3a4-12b7-11e6-81d1-346231156fa5.png)

![screenshot 2016-05-05 at 11 47 16 am](https://cloud.githubusercontent.com/assets/19208932/15048344/2a69453e-12b7-11e6-8f37-1d3176bd716c.png)

**Suggestion**: Have an email format validation feature in place to confirm correct format is entered in (i.e. name@domain.com)

**Comment by mjmilan** [comment]
I agree with Marlene that this is one the face of it madness, but I would like @tonysurma or @BillWagner to confirm that they actually want this fixed. I reason I hesitate is that someone has gone out of their way on the AccountController Tests to implement a test method called "ForgotPassword_InvalidUserName_DoesntSendEmail_But_RedirectsTo_PasswordResetRequestedView()" - and if you read that code it very definitely asserts precisely what Marlene (and I) considers to be a bug...
